app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/service/FurnitureService.kt

```diff
@@ -12,7 +12,7 @@ class FurnitureService {
 
     internal fun fetchFurnitures(furnitureName: String) : MutableLiveData<ArrayList<Furniture>> {
         //MVD is defined
-        var _furnitures = MutableLiveData<ArrayList<Furniture>>()
+        val _furnitures = MutableLiveData<ArrayList<Furniture>>()
```

Comment:
Change this variable to val since is never modified as explained here:
https://kotlinlang.org/docs/reference/typecasts.html#smart-casts